### ISSUE TITLE
Allow evaluating object detections on training set

### ISSUE THREAD

**Issue by fredzzhang** [fredzzhang/hicodet#4]
The [evaluation utility](https://github.com/fredzzhang/hicodet/tree/main/detections#evaluate-detections) only supports evaluation on test set. Add an argument to allow evaluation on training set
